README.md

```diff
@@ -51,7 +48,7 @@ Javascript React.js CSS
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-See the Math-Magician project <a href="https://majdalkilany.github.io/book Store/build"> here</a>
+See the Book-Store project <a href="https://bookstore-cuwm.onrender.com/"> here</a>
```

Comment:
- [ ] Well done working on your Readme however,  The link to the Live demo is not working kindly fix it. 

- [ ] The link to the MIT license is broken as well.

Reply:
Hi  @algerina  I am sure that the link is working Kindly check it again and it will show the react logo because it needs to merge to take the last updates